fixham/pkg/bootstrap/bootstrap.go

```diff
@@ -0,0 +1,54 @@
+package bootstrap
+
+import (
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/goyek/goyek/v2"
+	"github.com/goyek/x/boot"
+
+	"github.com/projectcalico/fixham/pkg/ctl"
+)
+
+var (
+	pkgName  *string
+	rootBind *string
+
+	bootstapPackageName string
+)
+
+// getPackageName returns the package name.
+func getPackageName() string {
+	return *pkgName
+}
+
+// NewGoBuildRunner returns a new instance of the GoBuildRunner.
+func NewGoBuildRunner() *ctl.GoBuildRunner {
+	return ctl.NewGoBuildRunner(getPackageName())
+}
+
+// Main is the entrypoint for the goyek build system.
+//
+// It defines common tasks that can be run.
+// It also sets the default task to run when no task is specified.
+//
+// If no default tasks are specified, the DefaultTask runs the "static-checks" task.
+func Main(packageName string, rootVolume string) {
+	flag.StringVar(&bootstapPackageName, "package", LookupEnv("PACKAGE_NAME", packageName), "The package name to use")
+	pkgName = &bootstapPackageName
+	if rootVolume == "" {
```

Comment:
I think we should use `envconfig` to handle reading environment variable configurations and avoid the need to scatter `os.LookupEnv` calls throughout the codebase, which become tricky to track down.

```go
type Config struct {
  PackageName string `envconfig:"PACKAGE_NAME"`
}
```

then

```go
cfg := Config{}
envconfig.Process("", &cfg)
```

e.g., https://github.com/projectcalico/calico/blob/616bad52431ff8aa4e2d6cfe9ec1a471045fad3e/libcalico-go/lib/apiconfig/apiconfig.go#L51-L65